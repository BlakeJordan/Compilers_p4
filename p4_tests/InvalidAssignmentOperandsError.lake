int a() {
    return 5;
}

int b() {
    return 6;
}

int f() {
    a = b;
    return a;
}
