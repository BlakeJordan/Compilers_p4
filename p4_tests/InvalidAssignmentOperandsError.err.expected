10,5: Invalid assignment operand
10,9: Invalid assignment operand
11,12: Bad return value
Type checking failed
